.travis.yml

```diff
@@ -6,11 +6,14 @@ env:
   global:
     - secure: "Vbdts9k+dvnzAmuRPXMDZLw2+s+4q0Ijdbx97rCBefrBbG+6elfR0ao60135QVAoiQxTR//Iz6ZZbsFfTTt9Qo4SVvulzepuLz4lDQmO1BTeKlAWyM650+uFDP39iRY+i6Z7w5RGZzw4F3Sjw8N4kcxdrkkJLH4p9ZQkA5uSkzg="
 
-php:
-  - 5.5
-  - 5.6
-  - 7.0
-  - hhvm
+matrix:
+  fast_finish: true
+  include:
+    - php: 5.5
+    - php: 5.6
+      env: WITH_CS=true
```

Comment:
@giorgiosironi 

I'm not entirely sure, but I don't think `fabpot/php-cs-fixer` supports fixing PHP7 issues yet, so I figured the best idea is to run it on the most recent version smaller than `7`.